bbc/simorgh
# WSTEAM1-788 - Live page SEO coverage datetime

Resolves https://jira.dev.bbc.co.uk/browse/WSTEAM1-788

Overall changes
======
- Adds `coverageStartTime` and `coverageEndTime` to SEO Schema metadata
- Updates the `LinkedData` component to take in these 2 new fields

Testing
======
1. Visit https://www.test.bbc.com/pidgin/live/c7p765ynk9qt
2. Inspect the SEO Schema for the page, using a tool such as DetailedSEO
3. Confirm that the `coverageStartTime` and `coverageEndTime` fields are set in the Schema

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
